app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div class="graph-table">
+    <div class="graph-table__title">
+      {{ title }}
+    </div>
+    <table class="graph-table__table">
+      <thead>
+        <th
+          v-for="({ year, percentageRelative, percentageAbsolute }, ix) in years"
+          :key="year"
+          class="graph-table__table--th"
+        >
+          <template v-if="ix !== 0">
+            <div class="graph-table__barchart--wrapper">
+              <div class="graph-table__barchart--wrapper-relative">
+                <div
+                  class="graph-table__barchart"
+                  :style="{ height: `${percentageAbsolute}%` }"
+                >
+                  <div
+                    class="graph-table__barchart--active"
+                    :style="{ height: `${percentageRelative}%` }"
+                  />
+                </div>
+              </div>
+              <span>{{ year }}</span>
+            </div>
+          </template>
+        </th>
+      </thead>
+      <tbody>
+        <tr
+          v-for="metric in metrics"
+          :key="metric.metric"
+          class="graph-table__table--tr"
+        >
+          <td
+            v-for="{ year } in years"
+            :key="year"
+            class="graph-table__table--td"
+          >
+            {{ metric[year] }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import { date, percent } from "lib/shared";
+
+const formatValues = (arr, prop, transform = x => x) =>
```

Comment:
Maybe you can extract this function in a new file `/lib/utils.js` what do you think?

Reply:
Mmmm, very specific function. Only applies to this component, in order to generate a particular object.
Besides, it has a internal dependency (`date` function), so better keep it right there.